widget_driver/example/lib/widgets/coffee_consumption/coffee_counter/coffee_counter_widget_driver.dart

```diff
@@ -33,15 +34,37 @@ class CoffeeCounterWidgetDriver extends WidgetDriver {
   @TestDriverDefaultValue('3')
   String get amountText => '${_consumptionService.counter}';
 
-  @TestDriverDefaultValue('Consume coffee')
-  String get consumeCoffeeButtonText => _localization.consumeCoffees;
+  @TestDriverDefaultValue('Consume coffee quick')
+  String get consumeCoffeeQuickButtonText => _localization.consumeCoffeeQuick;
+
+  @TestDriverDefaultValue('Consume coffee slow')
+  String get consumeCoffeeSlowButtonText =>
+      _isCurrentlyConsuming ? _localization.consumingCoffee : _localization.consumeCoffeeSlow;
 
   @TestDriverDefaultValue('Reset consumption')
   String get resetCoffeeButtonText => _localization.resetConsumedCoffees;
 
   @TestDriverDefaultValue()
-  void consumeCoffee() {
+  void consumeCoffeeQuick() {
+    _consumptionService.consumedOneCoffee();
+  }
+
+  /// Returns true if we could consume a coffee slow.
+  /// We cannot consume slow if we are already consuming a slow coffee.
+  @TestDriverDefaultFutureValue(false)
```

Comment:
Show cases how to use the new future annotations and also how to combine annotations together with documentation